pallets/rc-migrator/src/lib.rs

```diff
@@ -1237,7 +1245,7 @@ pub mod pallet {
 					Self::transition(MigrationStage::AccountsMigrationInit);
 				},
 				MigrationStage::AccountsMigrationInit => {
-					let weight = AccountsMigrator::<T>::obtain_rc_accounts();
+					let weight = AccountsMigrator::<T>::obtain_rc_accounts().saturating_add(AccountsMigrator::<T>::obtain_free_proxy_candidates());
```

Comment:
this stage getting quite big. can we check its not topp big already?

Reply:
may be separate stage for pure proxies? 
it actually can be at PureProxyInit stage